docs/algemeen.md

```diff
@@ -1,18 +1,18 @@
 # Regels
 
 ### **Artikel 1: Cheats (Straf-categorie: 9)**
-   - Een burger die gebruik maakt van software/hulpmiddelen van derde partijen om een voordeel in het spel te krijgen zullen worden bestraft.</br>
+   - Een burger die gebruik maakt van software/hulpmiddelen van derde partijen om een voordeel in het spel te krijgen zullen worden bestraft.<br>
```

Comment:
het is hier beter om <br /> te gebruiken

Reply:
Is gewoon hetzelfde.